runners/extensions-java/metrics/build.gradle

```diff
@@ -17,18 +17,20 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature()
+applyJavaNature(failOnWarning: true)
 
 description = "Apache Beam :: Runners :: Extensions Java :: Metrics"
 ext.summary = "Beam Runners Extensions Metrics provides implementations of runners core metrics APIs."
 
 
 dependencies {
     compile library.java.guava
+    compileOnly library.java.findbugs_annotations
     shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
     shadow library.java.jackson_databind
     shadow library.java.jackson_datatype_joda
-    shadow library.java.findbugs_annotations
```

Comment:
if I understand correctly you moved library.java.findbugs_annotations dep to only compile and testCompile scopes instead of all the scopes.

Reply:
Yes